.circleci/config.yml

```diff
@@ -376,3 +376,4 @@ workflows:
                 - main
     jobs:
       - build_docs
+
```

Comment:
just curious: why this change?

Reply:
Hm, just me poorly dealing with local changes, sorry.